spec/models/book_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Book, type: :model do
+  subject { build(:book, title: 'The Odd Sister', publishing_house: 'Opus Reader', language: 'English', year_of_publication: '2005', number_of_pages: '320', ISBN: '7256416783942') }
```

Comment:
don't override Faker attributes in case you don't need specific values for some of them

Reply:
why did your codestyle checks pass with that?

Reply:
done